code-of-conduct.rst

```diff
@@ -33,14 +33,14 @@ Encouraged Behaviour
 
 Examples of encouraged behaviour include:
 
-- Respectful discourse of disagreements around new features, changes, and
+- Discuss and disagree respectfully concerning new features, changes, and
```

Comment:
The COC requires us to disagree? :) I would use the neutral word "debate" perhaps.